homeassistant/components/prometheus/__init__.py

```diff
@@ -274,15 +286,25 @@ def handle_entity_registry_updated(
             self._remove_labelsets(metrics_entity_id)
 
     def _remove_labelsets(
-        self, entity_id: str, friendly_name: str | None = None
+        self,
+        entity_id: str,
+        friendly_name: str | None = None,
+        ignored_metric_names: set[str] | None = None,
     ) -> None:
         """Remove labelsets matching the given entity id from all metrics."""
+        if ignored_metric_names is None:
```

Comment:
The comment says "all metrics" which is no longer accurate, could you please adjust it?